spec/system/sponsor_additional_details_spec.rb

```diff
@@ -59,16 +57,25 @@
 
     it "validates other identity documents field on any submission" do
       navigate_to_additional_details
-      choose("I don't have any of these")
+      uam_enter_sponsor_identity_documents("I don't have any of these")
+
+      expect(page).to have_content("Enter your date of birth")
+    end
+
+    it "does not allow for empty nationality to be selected" do
+      navigate_to_nationality
+
       click_button("Continue")
 
-      expect(page).to have_content("Can you prove your identity?")
+      expect(page).to have_content("Error: You must select a valid nationality")
+    end
 
-      fill_in("unaccompanied-minor-no-identification-reason-field", with: "Hello")
+    it "shows an error box when no nationality is selected " do
+      navigate_to_nationality
 
       click_button("Continue")
 
-      expect(page).to have_content("Enter your date of birth")
+      expect(page).to have_content("There is a problem")
```

Comment:
I think this assertion can be moved into the above test as the arrange and act steps are the same.

Reply:
I agree, done it